.github/workflows/_mac-test-mps.yml

```diff
@@ -74,5 +75,17 @@ jobs:
           set -ex
           # TODO(https://github.com/pytorch/pytorch/issues/79293)
 
-          ${CONDA_RUN} --cwd test python3 test_mps.py -v
-          ${CONDA_RUN} --cwd test python3 test_metal.py -v
+          ${CONDA_RUN} python3 test/run_test.py --mps --verbose
+
+      - name: Get workflow job id
+        id: get-job-id
+        uses: ./.github/actions/get-workflow-job-id
+        if: always()
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Upload test artifacts
+        uses: ./.github/actions/upload-test-artifacts
+        if: always() && (steps.test.conclusion == 'success' || steps.test.conclusion == 'failure')
+        with:
```

Comment:
Now that M1 runner has S3 access with https://github.com/fairinternal/pytorch-gha-infra/pull/76, this is ok, but the setup is pretty much manual, so new runners might fail this step if they don't have the correct IAM profile attached.  It's safer to have `use-gha: true` here same as `_mac_test.yml` till we're ready